### ISSUE TITLE
Missing docstring error for class functions with docstrings

### ISSUE THREAD

**Issue by tayden-hakai** [AtomLinter/linter-pylama#18]
Getting `D102 Missing docstring in public method [pep257]` error for the following python class definition file. The error is on the \__init__ method definition line and also occurs when defining other class methods. Linting the file externally with pylama (via terminal) does not print the error. Tried changing settings to use both external and internal pylama executable.

    """Class definition for Djikstra algorithm node."""

    class Node:

        """Main class for Djikstra algorithm calculation."""

        def __init__(self, name, distances={}):
            """Init Djikstra node with name and distance information."""
            self.name = name
            self.distances = distances

**Comment by tayden-hakai** [comment]
Why is this closed? The docstring is defined and linter-pylama is saying it is not. This is a bug.

**Comment by gmist** [comment]
Should be fixed in v0.2.2
``` shell
> apm install linter-pylama@0.2.1
Installing linter-pylama@0.2.1 to /Users/serg/.atom/packages ✓
> ~/.atom/packages/linter-pylama/bin/pylama.py -l pep257 issue18.py
issue18.py:4:1: D211: No blank lines allowed before class docstring (found 1) [pep257]
issue18.py:8:1: D102: Missing docstring in public method [pep257]

> apm install linter-pylama@0.2.2
Installing linter-pylama@0.2.2 to /Users/serg/.atom/packages ✓
>  ~/.atom/packages/linter-pylama/bin/pylama.py -l pep257 issue18.py
issue18.py:4:1: D211: No blank lines allowed before class docstring (found 1) [pep257]
```
See 935b6c4
